flink-runtime/src/main/java/org/apache/flink/runtime/taskmanager/Task.java

```diff
@@ -359,21 +360,13 @@ public Task(
 
 		final String taskNameWithSubtaskAndId = taskNameWithSubtask + " (" + executionId + ')';
```

Comment:
Might remove this local var to use `taskNameWithSubtask + " (" + executionId + ')'` directly below.

Reply:
I would keep it to not mix concerns in one line.

Reply:
I could agree your way to keep it. :)

My previous concern is that `taskNameWithSubtaskAndId` was used in many places before, so it worths defining a local var. Now it is only used in one place and we could reduce one line to make task constructor short to remove it.
Another small consideration is avoiding final issue as above mentioned.